sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestingClientTestEnvironment.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    public class LoadTestingClientTestEnvironment : TestEnvironment
+    {
+        public string Endpoint => GetRecordedVariable("ENDPOINT");
```

Comment:
Best practice is to use a service specific name 
```suggestion
        public string Endpoint => GetRecordedVariable("LOADTESTSERVICE_ENDPOINT");
```

Reply:
Resolved